docarray/array/storage/redis/backend.py

```diff
@@ -19,8 +21,7 @@ class RedisConfig:
     n_dim: int
     host: str = field(default='localhost')
     port: int = field(default=6379)
-    index_name: str = field(default='idx')
-    flush: bool = field(default=False)
```

Comment:
why do we need to remove `flush`, user should still need flush right? if they want to clear the database

Reply:
`da.clear()` can remove this DocumentArray from database. If user wants to clear the whole database, I think they should excute `FLUSHDB` in Redis Server directly, not through an `flush` operation of one DocumentArray.

Reply:
@alaeddine-13 can you please advice about wether this is aligned with the rest of the DocArrays implementations?

Reply:
we don't have flush parameter in other backends. Let's just rely on `da.clear()` to clear document array. She also has a point about clearing whole database (creating a new DA should not affect other DAs in the database)